### ISSUE TITLE
parseExcludes in fractal is supported?

### ISSUE THREAD

**Issue by yizhihou2** [yajra/laravel-datatables#633]
### Summary of problem or feature request

I understand that laravel-datatables supports Fractal Transformer, as suggested in [this issue](https://github.com/yajra/laravel-datatables/issues/556) , but is there a way to use [Excluding Includes](http://fractal.thephpleague.com/transformers/) feature in laravel-datatables, and how?

Thanks!

**Comment by yajra** [comment]
It's not yet supported atm. Will try to implement this or if you can, PR's are welcome. Thanks!

**Comment by yajra** [comment]
I made a quick look on parseExcludes and I think it's not implemented on fractal package? 

```php
Call to undefined method League\Fractal\Manager::parseExcludes()
```

**Comment by yajra** [comment]
@yizhihou2 , I think fractal excludes is not yet implemented on that package and I got the error above when trying to implement it. Anyways, you can now extend the fractal manager by accessing it through the IoC `app('datatables.fractal')` and extend it as needed. PR #636 released on v6.15.1. Thanks!
